src/EventSubscriber/CentreonEventSubscriber.php

```diff
@@ -287,7 +287,15 @@ public function defineApiVersionInAttributes(RequestEvent $event): void
                 $requestApiVersion = $betaVersion;
             }
 
+            /**
+             * Used for the routing conditions.
+             * @todo We need to use an other name because after routing,
+             *       its value is overwritten by the value of the 'version' property from uri
```

Comment:
Couldn't you do it in this PR ?

Reply:
No because it requires to change all the routing rules of Centreon Web and modules.
This changes must be made in one shoot.